config/enabled.yml

```diff
@@ -540,3 +540,24 @@ Lint/UselessSetterCall:
 
 Lint/Void:
   Description: 'Possible use of operator/literal/variable in void context.'
+
+Layout/CommentIndentation:
+  Description: Indentation of comments.
+
+Layout/EmptyLinesAroundArguments:
+  Description: Keeps track of empty lines around method arguments.
+
+Layout/EmptyLinesAroundBeginBody:
+  Description: Keeps track of empty lines around begin-end bodies.
+  StyleGuide: "#empty-lines-around-bodies"
+
+Layout/FirstHashElementLineBreak:
+  Description: Checks for a line break before the first element in a multi-line hash.
+
+Layout/FirstArrayElementLineBreak:
+  Description: Checks for a line break before the first element in a multi-line array.
+
+Layout/IndentationConsistency:
```

Comment:
This one in particular is not to have enforced (we were actually doing a surprisingly good job here)